src/main/kotlin/backend/map/Vector.kt

```diff
@@ -3,5 +3,4 @@ package backend.map
 data class Vector(val x: Int, val y: Int) {
   operator fun plus(vector: Vector): Vector = Vector(this.x + vector.x, this.y + vector.y)
   operator fun minus(vector: Vector): Vector = Vector(this.x - vector.x, this.y - vector.y)
-
-}
+}
```

Comment:
nit: missing new line

Reply:
should there be at the end of a file?
not sure what is the convention in kotlin
btw, it doesn't matter, you can add it any time

Reply:
it's not Kotlin convention. https://stackoverflow.com/questions/6549452/new-line-at-the-end-of-source-code